### ISSUE TITLE
Problems parsing YAML block

### ISSUE THREAD

**Issue by shiroandy** [revolunet/sublimetext-markdown-preview#265]
I have the MarkdownPreview package installed and am unable to correctly parse the following YAML header when I build in HTML. 

```
---  
   title: Bla Bla Bla
   author: Peter and Paul     
   date: February 2015 
--- 

## Overview

More text 
```

I have no problems when i use pandoc to build a docx or pdf. I've also set the strip_yaml_front_matter option to true in user settings.

Does anyone know what's going on here?

############################################
Sublime 3 installed
Tested on Win 7 and Mac OSX

**Comment by facelessuser** [comment]
Yeah, I know what the problem is...white space :smile:.

You have trailing white space after your `---`; both the opening and the closing.

Yeah, yeah, the regex should probably be updated to allow for some trailing spaces, but for now, just don't put trailing spaces at the end of your `---`.

**Comment by facelessuser** [comment]
Turns out it was mainly the spaces after the closing `---`.  I have added a fix in pull request #264.

**Comment by shiroandy** [comment]
Many thanks!! That fixed it
